cmd/swctl/app/cmd_dependency.go

```diff
@@ -491,6 +491,14 @@ func InstallDocker(cli Cli, dockerVersion string) error {
 		color.Fprintln(cli.Out(), out)
 
 	}
+	errorGroup := postInstall(cli)
```

Comment:
Let's not overcomplicate it. PostInstall currently calls only one command (if that changes, refactoring will be necessary) so return only one error and remove commands for cycle and other related things.